doc/introduction/interfaces.rst

```diff
@@ -95,3 +104,4 @@ converting QNodes into both Keras and ``torch.nn`` layers:
     interfaces/numpy
     interfaces/torch
     interfaces/tf
+    interfaces/jax
```

Comment:
I don't think this module exists, right? If not this should be deleted

Reply:
Removed.

Reply:
Wait, does that mean there is no jax "module" in the API docs even though it is included above? At least I don't see it in the built. Is that not a bit confusing?

Reply:
Maria this was my fault. I got confused, thinking this was a TOC of Python modules... but its a TOC of ReST pages 😆 

I added it back in today when doing my final review.

But yes, there is no JAX module/JAX interface. JAX is only supported via backprop, where an interface is not required/needed. (Since interfaces 'unwrap' tensors and register custom gradients)